app/v1/controllers/AmendReliefInvestmentsController.scala

```diff
@@ -87,5 +95,12 @@ class AmendReliefInvestmentsController @Inject()(val authService: EnrolmentsAuth
     }
   }
 
+  private def auditSubmission(details: AmendReliefInvestmentsAuditDetail)
+                             (implicit hc: HeaderCarrier,
+                              ec: ExecutionContext) = {
+    val event = AuditEvent("amendReliefInvestments", "amend-relief-investments", details)
```

Comment:
The audit type and transaction name do not match the TXM document for this endpoint [here](https://confluence.tools.tax.service.gov.uk/pages/viewpage.action?pageId=202084824)